src/transformers/generation/logits_process.py

```diff
@@ -345,16 +345,48 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class TypicalLogitsWarper(LogitsWarper):
     r"""
-    [`LogitsWarper`] that performs typical decoding. See [Typical Decoding for Natural Language
-    Generation](https://arxiv.org/abs/2202.00666) for more information.
+    [`LogitsProcessor`] that applies typical warping techniques to the logits during text generation. It can be used to
+    control the randomness and diversity of generated text.
 
     Args:
-        mass (`float`):
-            Value of typical_p between 0 and 1 inclusive, defaults to 0.9.
-        filter_value (`float`, *optional*, defaults to `-float("Inf")`):
-            All filtered values will be set to this float value.
-        min_tokens_to_keep (`int`, *optional*, defaults to 1):
-            Minimum number of tokens that cannot be filtered.
+        temperature (:obj:`float`, optional, defaults to 1.0):
+            The temperature value controls the randomness of the generated text. Higher values (e.g., 1.5) make the
+            generated text more diverse, while lower values (e.g., 0.8) make it more focused and deterministic.
+
+        top_k (:obj:`int`, optional, defaults to 50):
+            The `top_k` parameter controls the number of most likely tokens to consider during sampling. It restricts
+            the sampling pool to the top `k` tokens with the highest probabilities at each step. Setting `top_k` to a
+            lower value (e.g., 20) will make the text generation more conservative.
+
+        top_p (:obj:`float`, optional, defaults to 1.0):
+            The `top_p` parameter, also known as nucleus sampling or probabilistic sampling, controls the cumulative
+            probability of the top tokens to consider during sampling. It defines the smallest set of tokens whose
+            cumulative probability exceeds `top_p`. Setting a lower `top_p` value (e.g., 0.9) will make the generation
+            more focused, as it includes only a narrower range of likely tokens.
```

Comment:
These are not the arguments to `TypicalLogitsWarper`. See the `__init__` function below